lib/DropdownMenu/index.js

```diff
@@ -47,16 +52,46 @@ class DropdownMenu extends Component {
         case 'link':
           return (
             <li className="dropdown__item" key={idKey}>
+              {item.active && (
+                <span className="dropdown-item__tick is-active">
+                  <Icon name="tick" size="micro" />
+                </span>
+              )}
               <a className="dropdown__link" href={item.href}>
                 {item.name}
               </a>
             </li>
           );
 
+        case 'withAdditional':
+          return (
+            <li className="dropdown__item" key={idKey}>
+              <button
+                className="dropdown__link dropdown__additional"
+                onClick={item.action}
+              >
+                {item.active && (
```

Comment:
It seems like the contents for `item.active` is the same across all cases. It might be worth assigning it before the case and adding it in where required e.g.

```
static makeItems(items) {
  const activeContents = (
    <span className="etc">...</span>
  )

  // you can then add this into multiple cases.
}
```